docs/data/data-grid/aggregation/aggregation.md

```diff
@@ -4,15 +4,275 @@ title: Data Grid - Aggregation
 
 # Data Grid - Aggregation [<span class="plan-premium"></span>](https://mui.com/store/items/mui-x-premium/)
 
-<p class="description">Apply aggregation function to populate the group row with values.</p>
+<p class="description">Use aggregation functions to combine your row values.</p>
 
-> ⚠️ This feature isn't implemented yet. It's coming.
->
-> 👍 Upvote [issue #213](https://github.com/mui/mui-x/issues/213) if you want to see it land faster.
+The aggregation can be modified through the grid interface by opening the column menu and selecting an item on the _Aggregation_ select.
 
-When grouping, you will be able to apply an aggregation function to populate the group row with values.
+The aggregated values will be rendered in a footer row at the bottom of the grid.
+
+:::warning
+⚠️ The footer row will be pinned at the bottom of the grid once [#1251](https://github.com/mui/mui-x/issues/1251) is ready.
+:::
+
+{{"demo": "AggregationBasic.js", "bg": "inline", "defaultCodeOpen": false}}
+
+## Pass aggregation to the grid
+
+### Structure of the model
+
+The aggregation model is a lookup where the keys are the columns, and the values are aggregation items, containing the functions defining the aggregation behavior, and where the aggregated values should be displayed.
```

Comment:
I had a similar problem in https://mui.com/x/react-data-grid/editing/#controlled-mode

"lookup" is kind of a term we use internally. In JavaScript it's called "object". In other languages, "dictionary" is used, since it has a value for each key.

Reply:
Ah ok that makes sense now, given that what follows is a description of key-value pairs. In that case I think we should go with the standard term.

```suggestion
The aggregation model is an object.
The keys correspond to the columns, and the values are aggregation items.
The values contain the functions that define the aggregation behavior, and are where the aggregated values are displayed.
```